/** 
 * Syntax Related
*/

// console.log("Hello

// Runtime exceptions

/**
let x=5
console.log(x.toUpperCase())
*/

/**
 * Logical
 

let num=5
for (i=-3;i<5;i++){
    console.log(num/i)
}
*/

try{
    let obj ={
        name: "Shivam"
    }
    console.log(obj.name)
}catch(err){
    console.log("Exception handler")
}finally {
    console.log("I will always executed")
}